layouts/blog/list.html

```diff
@@ -1,35 +1,37 @@
 {{ define "main" }}
-    <div class="md:flex">
+    <div class="container mx-auto px-8 py-8 md:px-0">
+        <div class="md:flex">
 
-        <div class="md:w-3/12 pr-8">
-            {{ partial "blog/sidebar.html" . }}
-        </div>
+            <div class="md:w-3/12 pr-8">
+                {{ partial "blog/sidebar.html" . }}
+            </div>
 
-        <div class="md:w-7/12">
-            {{ $paginator := .Paginate (where .Pages "Type" "blog").ByDate.Reverse }}
-            {{ range $paginator.Pages }}
-                <article class="mb-8 pb-8 border-b border-gray-200">
-                    <h2 class="no-anchor"><a href="{{ .Permalink }}">{{ .Title }}</a></h2>
-                    <div class="flex items-center">
-                        <span>
-                            {{ partial "blog/authors.html" (dict "context" . "authors" .Params.authors ) }}
-                        </span>
-                        <time>
-                            {{ .Date.Format "Monday, Jan 2, 2006" }}
-                        </time>
-                    </div>
-                    <section>
-                        {{ partial "blog/poster" . }}
-                        {{ partial "blog/summary" . }}
-                    </section>
-                </article>
-            {{ end }}
+            <div class="md:w-7/12">
+                {{ $paginator := .Paginate (where .Pages "Type" "blog").ByDate.Reverse }}
+                {{ range $paginator.Pages }}
+                    <article class="mb-8 pb-8 border-b border-gray-200">
+                        <h2><a href="{{ .Permalink }}">{{ .Title }}</a></h2>
```

Comment:
Should we keep the `no-anchor` class here? It doesn't seem that useful to be able to get an anchor link inside the list of blog posts, especially since the page will be constantly changing as we add new posts.

Reply:
That makes sense, yeah; will add it. I do wonder if there's a better approach to these anchors, though.